config/pipeline.yaml

```diff
@@ -1868,6 +1884,10 @@ build_configs:
     tree: arnd
     branch: 'to-build'
 
+  arm64:
```

Comment:
This should go a few lines up ("arm64" < "arnd") -- same comment for the tree entry.

Reply:
Done.